packages/clay-toolbar/src/Input.tsx

```diff
@@ -0,0 +1,26 @@
+/**
+ * SPDX-FileCopyrightText: © 2020 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {ClayInput} from '@clayui/form';
+import classNames from 'classnames';
+import React from 'react';
+
+const Input: React.FunctionComponent<React.HTMLAttributes<
+	HTMLInputElement
+>> = ({className, ...otherProps}) => (
+	<ClayInput.Group>
+		<ClayInput.GroupItem>
+			<ClayInput
+				className={classNames(className, 'form-control-inline')}
```

Comment:
The class `form-control-inline` doesn't exist in Bootstrap or Clay CSS. The class `form-control` should suffice.